src/app/controllers/OrderController.ts

```diff
@@ -365,9 +435,12 @@ export class OrderController {
       let mutationString = `
                 query getOrderDetails ($orderId: String!){
                     getOrderDetails(order_id: $orderId) {
-                        _id
+                      ...OrderFragment
                     }
                 }
+                ` 
```

Comment:
Don't you already have these isnt it a duplicate tick?

Reply:
Where do we have fragments? What is this duplicating?